internal/app/connectconformance/client_runner.go

```diff
@@ -225,3 +228,15 @@ func (c *clientProcessRunner) consumeOutput() {
 		action(resp.TestName, resp, nil)
 	}
 }
+
+type errFailedToGetResult struct {
```

Comment:
This marker type is used to avoid printing "Receive response for ..." verbose messages. When the test runner sees this as the cause of failure for a test case, it knows that it didn't _actually_ receive any response.